service/storage_service.hh

```diff
@@ -291,13 +291,18 @@ private:
     future<> shutdown_protocol_servers();
 
     struct replacement_info {
+        locator::host_id replace_host_id;
+        gms::inet_address replace_address;
```

Comment:
This is inside replacement_info, just call it address like other members without the replace prefix?

Reply:
right, done this way in next version